<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserBox</class>
 <widget class="QDialog" name="UserBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>210</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(214, 199, 170);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Phone Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Last Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
